addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -990,6 +990,14 @@ registry.anchorSlide = publicWidget.Widget.extend({
             return;
         }
         var $anchor = $(hash);
+        if (hash === '#top' && !$anchor.length) {
+            // The header has been removed and there is no element with an id
+            // set to "top" in the DOM. In this case, the anchor targets
+            // wrapwrap by default. Indeed, it an element that is always
+            // present in the DOM and that is at the top of the document.
+            $anchor = $('#wrapwrap');
+            $anchor.attr('data-anchor', true);
+        }
```

Comment:
@loco-odoo No, do as we said in DM yesterday:
- If hash is `top` or `bottom`, don't even search for the top/bottom anchors, directly call the `dom.scrollTo` util by giving it "top" or "bottom" (a string)
- Patch the dom util to accept a string ("top" or "bottom") too
- I would not handle the #wrapwrap case anywhere, we should encourage to use "#top" which is more fancy
- Patch the editor so that the "top" and "bottom" choices are always suggested as internal link anchors during a link edition (even if the header/footer is not there).